src/material/stepper/stepper-module.ts

```diff
@@ -34,8 +34,6 @@ import {MatStepContent} from './step-content';
   ],
   exports: [
     MatCommonModule,
-    MatHorizontalStepper,
-    MatVerticalStepper,
```

Comment:
Technically this change shouldn't be necessary, but for some reason keeping the old steppers in the `exports` causes a null pointer exception in the ViewEngine compiler.